Formula/pyroscope.rb

```diff
@@ -0,0 +1,60 @@
+class Pyroscope < Formula
+  desc "Open source continuous profiling software"
+  homepage "https://pyroscope.io"
+  url "https://dl.pyroscope.io/release/pyroscope-0.5.1-source.tar.gz"
+  sha256 "a5f138a04a7a3b31c9e693370d9c214e3a6724fe14fbe123f8b7adc4ae28aff3"
+  license "Apache-2.0"
+  head "https://github.com/pyroscope-io/pyroscope.git", branch: "main"
+
+  depends_on "go" => :build
+  depends_on "node" => :build
+  depends_on "rust" => :build
+  depends_on "yarn" => :build
+  depends_on "zstd" => :build
+
+  on_linux do
+    depends_on "libunwind"
+    depends_on "php"
+    depends_on "gcc"
+  end
+
+  fails_with gcc: "5"
+
+  def install
+    system "make", "build-third-party-dependencies"
```

Comment:
What are the third party dependencies here?

Reply:
> What are the third party dependencies here?

pyroscope depends on 3 projects — phpspy, py-spy and rbspy  — these allow for profiling of php, python and ruby scripts respectively. py-spy and rbspy  are available via homebrew, but pyroscope uses forks of these projects so we can’t use versions provided by homebrew

Reply:
I think this is unsuitable for packaging in homebrew-core, then. See https://docs.brew.sh/Acceptable-Formulae#stuff-that-requires-vendored-versions-of-homebrew-formulae.

You may wish to host this formula in your own tap instead.